lib/V8/v8-globals.h

```diff
@@ -539,9 +539,6 @@ struct TRI_v8_global_t {
   /// @brief replicated log template
   v8::Persistent<v8::ObjectTemplate> VocbaseReplicatedLogTempl;
```

Comment:
Is it used?

Reply:
Yes in our tests, we expose access to those logs through V8.

Reply:
In : arangod/V8Server/v8-replicated-logs.cpp

Reply:
Thanks, I just tried to find it in devel, but it's probably new usage